operators/aqua/2022.4.1/bundle.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
```

Comment:
Hi @yossig-aquasec,

In this PR you are adding two bundles. 
Could you please create one PR for each distribution?

Reply:
@camilamacedo86 yes, done